lib/diagnostic.rb

```diff
@@ -7,20 +7,25 @@
 #
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
+ruby lib/diagnostic.rb
 
 Diagnostic.script = 'your response here'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
+pry
 
 Diagnostic.repl = 'your response here'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
+def star_wars_episode_viii
+  # p 'star_wars_episode_viii'
+end
```

Comment:
This is a method we just wanted a variable, but you did get the syntax correct so I'll give you credit.